lib/active_model_serializers.rb

```diff
@@ -3,6 +3,11 @@
 require 'active_support/core_ext/object/with_options'
 require 'active_support/core_ext/string/inflections'
 require 'active_support/json'
+
+require 'active_model_serializers/model_mixin'
```

Comment:
proposal: `active_model_serializers/serializable`, or `active_model_serializers/serializer_support` similar to earlier apis [0.8](https://github.com/rails-api/active_model_serializers/tree/0-8-stable#support-for-poros-and-other-orms) [0.9](https://github.com/rails-api/active_model_serializers/blob/0-9-stable/lib/active_model/serializer_support.rb)